src/main/java/uk/ac/cam/cl/dtg/segue/api/AdminFacade.java

```diff
@@ -1127,7 +1126,6 @@ public Response getDiagnostics(@Context final Request request, @Context final Ht
   @Path("/sync_external_accounts")
   @Operation(summary = "Trigger an update for external providers where account details have changed.")
   public Response syncExternalAccounts(@Context final HttpServletRequest httpServletRequest) {
-    //TODO - automate this with Quartz, then review if this is still necessary?
```

Comment:
Do we have any plans to do this? Is there a ticket for this in the backlog? Or do we think that the Quartz suggestion is not a good one?

Reply:
As mentioned in the full list, there is already a scheduled job that calls the synchronisation method. I'm not sure why the TODO wasn't cleared at the time but that's been in place since 2021.